aspnetcore/signalr/dotnet-client/sample/SignalRChatClient/packages.config

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.AspNetCore.Connections.Abstractions" version="2.1.0-preview2-final" targetFramework="net461" />
+  <package id="Microsoft.AspNetCore.Http.Connections.Client" version="1.0.0-preview2-final" targetFramework="net461" />
+  <package id="Microsoft.AspNetCore.Http.Connections.Common" version="1.0.0-preview2-final" targetFramework="net461" />
+  <package id="Microsoft.AspNetCore.Http.Features" version="2.1.0-preview2-final" targetFramework="net461" />
+  <package id="Microsoft.AspNetCore.SignalR.Client" version="1.0.0-preview2-final" targetFramework="net461" />
+  <package id="Microsoft.AspNetCore.SignalR.Client.Core" version="1.0.0-preview2-final" targetFramework="net461" />
+  <package id="Microsoft.AspNetCore.SignalR.Common" version="1.0.0-preview2-final" targetFramework="net461" />
```

Comment:
What is the plan with updating to RC1?

Reply:
@JamesNK  Is there a MyGet for the latest client package? The server components are all RC1 but the client package wasn't updated so this is the latest client pkg I have.

Reply:
RC1 final isn't publicly available yet but will be soon.

Reply:
@JamesNK I meant an internal one. For employees.

Reply:
The feeds are public, but you also don't really need a feed. If you install an RC build from https://github.com/dotnet/versions/tree/master/build-info/dotnet/product/cli/release/2.1 the packages are all in the offline cache automatically.

Reply:
@anurse  Hmmmm. That is where I installed from for 2.1 RC. The server package built into VS seems life. For the client package I don't see anything RC available. I must be missing something.

Reply:
Ah, yeah, the client libraries aren't in that package actually. I emailed some info to you.

Reply:
@anurse Thanks!